compose.yml

```diff
@@ -119,6 +122,14 @@ services:
             - MATOMO_DATABASE_PASSWORD=$MW_DB_INSTALLDB_PASS
         volumes:
             - ./_data/matomo:/var/www/html
+        labels:
+            - "traefik.enable=true"
+            - "traefik.http.routers.matomo.rule=Host(`bugsigdb-staging.wikiworks.com`) && PathPrefix(`/matomo`)"
```

Comment:
We need unique names for the routers. In other case several stacks won't work on one computer. please use ${TRAEFIK_STACK_PREFIX?Variable not set} or it is better to use the `${COMPOSE_PROJECT_NAME}`  already defined env variable and get rid of the TRAEFIK_STACK_PREFIX (replace it with `${COMPOSE_PROJECT_NAME}` )

Reply:
ok